server/tests/tables_structs/alert_struct.js

```diff
@@ -0,0 +1,10 @@
+const AlertStruct = function(table) {
+	table.increments();
+	table.integer('person_id');
+	table.string('address');
+	table.integer('radius');
+	table.binary('geom');
```

Comment:
maybe alerts aren't updated so the error we had with plan crawling never happens here, but should be changed to `table.specificType('geom', 'GEOMETRY');`